update entra-previews links

BenBagBag opened the pull request:
## Purpose
<!-- Describe the intention of the changes being proposed. What problem does it solve or functionality does it add? -->
* Updates link/link names for old entra-preview links. NB: this **does not** update other wording. Expect sentences like "get a CIAM tenant and use this Entra tutorial <link>". This will hopefully prevent messy merge issues later on.

## Does this introduce a breaking change?
<!-- Mark one with an "x". -->
```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

<!-- Please check the one that applies to this PR using "x". -->
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
<!-- Add steps to run the tests suite and/or manually test -->
```
```

## What to Check
Verify that the following are valid
* ...

## Other Information
<!-- Add any other helpful information that may be needed here. -->

iambmelt commented on 1-Authentication/0-sign-in-vanillajs/README.md:
```diff
@@ -63,8 +63,8 @@ Here you'll learn about [ID Tokens](https://docs.microsoft.com/azure/active-dire
 * A modern web browser.
-* An **Microsoft Entra External ID** tenant. For more information, see: [How to get an external tenant](https://github.com/microsoft/entra-previews/blob/PP2/docs/1-Create-a-CIAM-tenant.md)
-* A user account with permissions to create applications in your **Microsoft Entra External ID** tenant.
+* An **external Microsoft Entra ID** tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/en-us/entra/fundamentals/create-new-tenant)
```
I'll defer to @kengaderdus to say whether "external Microsoft Entra ID tenant" or "Microsoft Entra External ID tenant" is preferred

iambmelt replied:
Oh nevermind, I see Katherine asked for it this way 👍

kengaderdus replied:
Just say _external tenant_, here and in all other instances as you already told the user the product name, that's _Microsoft Entra External ID._

iambmelt reviewed (approved):
LGTM, let's also get a review from @kengaderdus before merging

kengaderdus commented on 1-Authentication/0-sign-in-vanillajs/README.md:
```diff
@@ -36,7 +36,7 @@ extensions:
 ## Overview
 
-This sample demonstrates a vanilla JavaScript single-page application (SPA) that lets users authenticate to [Microsoft Entra External ID](https://github.com/microsoft/entra-previews/blob/PP2/docs/1-Create-a-CIAM-tenant.md) (Microsoft Entra External ID) using the [Microsoft Authentication Library for JavaScript](https://github.com/AzureAD/microsoft-authentication-library-for-js) (MSAL.js). 
+This sample demonstrates a vanilla JavaScript single-page application (SPA) that lets users authenticate to [Microsoft Entra External ID](https://learn.microsoft.com/en-us/entra/fundamentals/create-new-tenant) using the [Microsoft Authentication Library for JavaScript](https://github.com/AzureAD/microsoft-authentication-library-for-js) (MSAL.js). 
```
```suggestion
This sample demonstrates a vanilla JavaScript single-page application (SPA) that lets users authenticate to [Microsoft Entra External ID](https://learn.microsoft.com/entra/external-id/external-identities-overview) using the [Microsoft Authentication Library for JavaScript](https://github.com/AzureAD/microsoft-authentication-library-for-js) (MSAL.js). 
```

kengaderdus replied:
@BenBagBag 

For links, remove the locale. For examples use `https://learn.microsoft.com/entra/external-id/external-identities-overview` instead of `https://learn.microsoft.com/en-us/entra/external-id/external-identities-overview`.

kengaderdus commented on 1-Authentication/0-sign-in-vanillajs/README.md:
```diff
@@ -63,8 +63,8 @@ Here you'll learn about [ID Tokens](https://docs.microsoft.com/azure/active-dire
 * A modern web browser.
-* An **Microsoft Entra External ID** tenant. For more information, see: [How to get an external tenant](https://github.com/microsoft/entra-previews/blob/PP2/docs/1-Create-a-CIAM-tenant.md)
-* A user account with permissions to create applications in your **Microsoft Entra External ID** tenant.
+* An **external Microsoft Entra ID** tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/en-us/entra/fundamentals/create-new-tenant)
```
```suggestion
* An external tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

kengaderdus reviewed (changes requested):
@BenBagBag
Here's a summary of my comments:
- In the links, remove the locale.
- Refer to the CIAM tenant as external tenant, but not `CIAM tenant` or `external Microsoft Entra ID tenant` or `Microsoft Entra External ID tenant`

BenBagBag commented:
@kengaderdus I have updated the links.

For the wording, your suggestion does not match up with the request in the original ticket (I split the link changes out into a separate ticket, hence different PR): https://identitydivision.visualstudio.com/Engineering/_workitems/edit/2960843?src=WorkItemMention&src-action=artifact_link

Can you consult with @kleggMS and @henrymbuguakiarie to get agreement on wording? Thank you!

kengaderdus commented:
@BenBagBag 

What's not matching? Can you mention one instance or example.

What I said is, if you refer to the product name, use _Microsoft Entra External ID_, if you're referring to the CIAM tenant, use _external tenant_.

BenBagBag commented:
@kengaderdus Ah, that makes sense, thanks for clarifying. I've updated it, please re-review.

CC: @kleggMS

kengaderdus commented on 1-Authentication/0-sign-in-vanillajs/README.md:
```diff
@@ -36,7 +36,7 @@ extensions:
 ## Overview
 
-This sample demonstrates a vanilla JavaScript single-page application (SPA) that lets users authenticate to [Microsoft Entra External ID](https://github.com/microsoft/entra-previews/blob/PP2/docs/1-Create-a-CIAM-tenant.md) (Microsoft Entra External ID) using the [Microsoft Authentication Library for JavaScript](https://github.com/AzureAD/microsoft-authentication-library-for-js) (MSAL.js). 
+This sample demonstrates a vanilla JavaScript single-page application (SPA) that lets users authenticate to [Microsoft Entra External ID](https://learn.microsoft.com/entra/fundamentals/create-new-tenant) using the [Microsoft Authentication Library for JavaScript](https://github.com/AzureAD/microsoft-authentication-library-for-js) (MSAL.js). 
```
```suggestion
This sample demonstrates a vanilla JavaScript single-page application (SPA) that lets users authenticate to [Microsoft Entra External ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal) using the [Microsoft Authentication Library for JavaScript](https://github.com/AzureAD/microsoft-authentication-library-for-js) (MSAL.js). 
```

kengaderdus commented on 1-Authentication/0-sign-in-vanillajs/README.md:
```diff
@@ -63,8 +63,8 @@ Here you'll learn about [ID Tokens](https://docs.microsoft.com/azure/active-dire
 * A modern web browser.
-* An **Microsoft Entra External ID** tenant. For more information, see: [How to get an external tenant](https://github.com/microsoft/entra-previews/blob/PP2/docs/1-Create-a-CIAM-tenant.md)
-* A user account with permissions to create applications in your **Microsoft Entra External ID** tenant.
+* An external tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/fundamentals/create-new-tenant)
```
```suggestion
* An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

kengaderdus commented on 1-Authentication/1-sign-in-react/README.md:
```diff
@@ -62,8 +62,8 @@ Here you'll learn how to [sign-in](https://docs.microsoft.com/azure/active-direc
 * [VS Code Azure Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack) extension is recommended for interacting with Azure through VS Code Interface.
-* An **Microsoft Entra External ID** tenant. For more information, see: [How to get an external tenant](https://github.com/microsoft/entra-previews/blob/PP2/docs/1-Create-a-CIAM-tenant.md)
-* A user account in your **Microsoft Entra External ID** tenant.
+ * An external tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/fundamentals/create-new-tenant)
```
```suggestion
 * An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID]https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

kengaderdus commented on 1-Authentication/2-sign-in-angular/README.md:
```diff
@@ -62,7 +62,7 @@ Here you'll learn about [ID Tokens](https://docs.microsoft.com/azure/active-dire
 * [VS Code Azure Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack) extension is recommended for interacting with Azure through VS Code Interface.
 * A modern web browser.
-* An **CIAM** tenant. For more information, see: [How to get an external tenant](https://github.com/microsoft/entra-previews/blob/PP2/docs/1-Create-a-CIAM-tenant.md)
+* An external tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/fundamentals/create-new-tenant)
```
```suggestion
* An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

kengaderdus commented on 1-Authentication/3-sign-in-electron/README.md:
```diff
@@ -68,8 +68,8 @@ Here you'll learn how to sign-in users and acquire [ID tokens](https://docs.micr
 * A modern web browser.
-* An **Microsoft Entra External ID** tenant. For more information, see: [How to get an external tenant](https://github.com/microsoft/entra-previews/blob/PP2/docs/1-Create-a-CIAM-tenant.md)
-* A user account in your **Microsoft Entra External ID** tenant.
+ * An external tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/fundamentals/create-new-tenant)
```
```suggestion
 * An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

kengaderdus commented on 1-Authentication/4-sign-in-device-code/README.md:
```diff
@@ -54,8 +54,8 @@ This sample demonstrates a Node headless application that authenticates users wi
 
-* An **Microsoft Entra External ID** tenant. For more information, see: [How to get an external tenant](https://github.com/microsoft/entra-previews/blob/PP2/docs/1-Create-a-CIAM-tenant.md)
-* A user account in your **Microsoft Entra External ID** tenant.
+ * An external tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/fundamentals/create-new-tenant)
```
```suggestion
 * An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

kengaderdus commented on 1-Authentication/6-sign-in-node-cli-app/README.md:
```diff
@@ -61,8 +61,8 @@ Here you'll learn how to sign-in users and acquire [ID tokens](https://docs.micr
 * A modern web browser.
-* An **Microsoft Entra External ID** tenant. For more information, see: [How to get an external tenant](https://github.com/microsoft/entra-previews/blob/PP2/docs/1-Create-a-CIAM-tenant.md)
-* A user account in your **Microsoft Entra External ID** tenant.
+ * An external tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/fundamentals/create-new-tenant)
```
```suggestion
 * An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

kengaderdus commented on 2-Authorization/0-call-api-vanillajs/README.md:
```diff
@@ -70,8 +70,8 @@ Here you'll learn about [access tokens](https://docs.microsoft.com/azure/active-
 * A modern web browser.
-* An **Microsoft Entra External ID** tenant. For more information, see: [How to get an external tenant](https://github.com/microsoft/entra-previews/blob/PP2/docs/1-Create-a-CIAM-tenant.md)
-* A user account in your **Microsoft Entra External ID** tenant.
+ * An external tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/fundamentals/create-new-tenant)
```
```suggestion
 * An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

kengaderdus commented on 2-Authorization/1-call-api-react/README.md:
```diff
@@ -67,8 +67,8 @@ Here you'll learn about [access tokens](https://docs.microsoft.com/azure/active-
 * Either [Visual Studio](https://visualstudio.microsoft.com/downloads/) or [Visual Studio Code](https://code.visualstudio.com/download) and [.NET Core SDK](https://www.microsoft.com/net/learn/get-started)
-* An **Microsoft Entra External ID** tenant. For more information, see: [How to get an external tenant](https://github.com/microsoft/entra-previews/blob/PP2/docs/1-Create-a-CIAM-tenant.md)
-* A user account in your **Microsoft Entra External ID** tenant.
+ * An external tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/fundamentals/create-new-tenant)
```
```suggestion
 * An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

kengaderdus commented on 2-Authorization/2-call-api-angular/README.md:
```diff
@@ -69,8 +69,8 @@ Here you'll learn about [access tokens](https://docs.microsoft.com/azure/active-
 * Either [Visual Studio](https://visualstudio.microsoft.com/downloads/) or [Visual Studio Code](https://code.visualstudio.com/download) and [.NET Core SDK](https://www.microsoft.com/net/learn/get-started)
-* An **Microsoft Entra External ID** tenant. For more information, see: [How to get an external tenant](https://github.com/microsoft/entra-previews/blob/PP2/docs/1-Create-a-CIAM-tenant.md)
-* A user account with permissions to create applications in your **Microsoft Entra External ID** tenant.
+ * An external tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/fundamentals/create-new-tenant)
```
```suggestion
* An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

kengaderdus commented on 2-Authorization/3-call-api-node-daemon/README.md:
```diff
@@ -67,8 +67,8 @@ Here you'll learn about [access tokens](https://docs.microsoft.com/azure/active-
 * Either [Visual Studio](https://visualstudio.microsoft.com/downloads/) or [Visual Studio Code](https://code.visualstudio.com/download) and [.NET Core SDK](https://www.microsoft.com/net/learn/get-started)
-* An **Microsoft Entra External ID** tenant. For more information, see: [How to get an external tenant](https://github.com/microsoft/entra-previews/blob/PP2/docs/1-Create-a-CIAM-tenant.md)
-* A user account in your **Microsoft Entra External ID** tenant.
+ * An external tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/fundamentals/create-new-tenant)
```
```suggestion
* An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

kengaderdus commented on 2-Authorization/4-call-api-express/README.md:
```diff
@@ -67,8 +67,8 @@ Here you'll learn about [access tokens](https://docs.microsoft.com/azure/active-
 * Either [Visual Studio](https://visualstudio.microsoft.com/downloads/) or [Visual Studio Code](https://code.visualstudio.com/download) and [.NET Core SDK](https://www.microsoft.com/net/learn/get-started)
-* An **Microsoft Entra External ID** tenant. For more information, see: [How to get an external tenant](https://github.com/microsoft/entra-previews/blob/PP2/docs/1-Create-a-CIAM-tenant.md)
-* A user account in your **Microsoft Entra External ID** tenant.
+* An external tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/fundamentals/create-new-tenant)
```
```suggestion
* An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal)
```

kengaderdus reviewed (changes requested):
@BenBagBag 
I noticed the link for creating an external tenant was pointing to the wrong doc. So, I've update all  instances of instances of An external tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/fundamentals/create-new-tenant) to  An external tenant. For more information, see: [Create a new external tenant in Microsoft Entra ID](https://learn.microsoft.com/entra/external-id/customers/how-to-create-external-tenant-portal).

Once this change is made, then everything is good for me.
